tests/unit/stream_alert_rule_processor/test_rules_engine.py

```diff
@@ -281,6 +281,57 @@ def web_server(rec):  # pylint: disable=unused-variable
         assert_equal(alerts[0].rule_name, 'web_server')
         assert_equal(alerts[1].rule_name, 'data_location')
 
+    def test_process_subkeys_false_none_value(self):
+        """Rules Engine - Req Subkeys with False and None values"""
+        @rule(logs=['test_log_type_json_nested'],
+              outputs=['s3:sample_bucket'],
+              req_subkeys={'data': ['value']})
+        def value_false(rec):  # pylint: disable=unused-variable
+            return rec['data']['value'] == False
```

Comment:
it looks like this is the line that is causing CI (pylint) to fail - changing to the following should fix:

`return rec['data']['value'] is False`

Reply:
Ah, should have run pylint locally. Cheers, will fix this.